packaging/hipify-clang.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(hipify-clang)
+
+install(PROGRAMS @PROJECT_BINARY_DIR@/hipify-clang/hipify-clang DESTINATION bin)
+
+#############################
+# Packaging steps
+#############################
+set(CPACK_SET_DESTDIR TRUE)
+set(CPACK_INSTALL_PREFIX "/opt/rocm/hip")
```

Comment:
What would be the CPACK_INSTALL_PREFIX for hipify-clang?

Reply:
1. `hipify-clang` is a multi-platform tool, but as long as `HIP` itself and its cmake building/installing system are not multi-platform yet, we may not, I guess, add Windows support for hipify-clang at a packaging step for now.
2. As for `CPACK_INSTALL_PREFIX` for `hipify-clang` for `UNIX` platforms, well, it is a question. As far as hipify-clang is going to be shipped with `hip-clang`, not with `HIP`, what is CPACK_INSTALL_PREFIX for `hip-clang`?

Reply:
> What would be the CPACK_INSTALL_PREFIX for hipify-clang?
1. `hipify-clang` is a multi-platform tool, but as long as `HIP` itself and its cmake building/installing system are not multi-platform yet, we may not, I guess, add `Windows` support for `hipify-clang` at a packaging step for now.
2. As for `CPACK_INSTALL_PREFIX` for `hipify-clang` for `UNIX` platforms, well, it is a question. As far as `hipify-clang` is going to be shipped with `hip-clang`, not with `HIP`, what is `CPACK_INSTALL_PREFIX` for `hip-clang`?

Reply:
1. Sure..
2. Where can I find the CPACK_INSTALL_PREFIX for hip-clang?